File: Практика_самостоятельная работа/C++/2.19.cpp
#include <iostream>
#include <cmath>

using namespace std;

double calculate_x(double a, double b) {
    return 2 / (a * a + 25) + (b / (1 / sqrt(b)) + a + b / 2);
}

double calculate_y(double a, double b) {
    return (abs(a) + 2 * sin(b)) / (5.5 * a);
}

int main() {
    double a, b, x, y;

    cout << "Введите a: ";
    cin >> a;
    cout << "Введите b: ";
    cin >> b;

    x = calculate_x(a, b);
    y = calculate_y(a, b);

    cout << "x = " << x << endl;
    cout << "y = " << y << endl;

    return 0;
}

File: Практика_самостоятельная работа/Python/2.19.py
import math

# Функция для вычисления x
def calculate_x(a, b):
    return 2 / (a**2 + 25) + (b / (1 / math.sqrt(b)) + a + b / 2)

# Функция для вычисления y
def calculate_y(a, b):
    return (abs(a) + 2 * math.sin(b)) / (5.5 * a)

# Ввод значений a и b
a = float(input("Введите a: "))
b = float(input("Введите b: "))

# Вычисление и вывод результатов
x = calculate_x(a, b)
y = calculate_y(a, b)
print("x =", x)
print("y =", y)
